profit/fit.py

```diff
@@ -65,11 +56,10 @@ def learn(self, plot_searching_phase=True):
 
         # First runs
         nfirst = 3
-        for kfirst in range(nfirst):
-            is_al = np.isnan(self.x[kfirst])
-            al_value = np.random.random(sum(is_al))
-            self.x[kfirst, is_al] = al_value
-            self.update_run(kfirst, al_value)
+        if nfirst < self.config['ntrain']:
+            nfirst = self.config['ntrain']
```

Comment:
`nfirst` must be less than `ntrain` since we want to select the `nfirst` points randomly and thereafter, with the rest of the `ntrain` points, do active learning.

Reply:
Within `tests/config` most configurations have an `ActiveLearning` variable, but `ntrain: 2`. These lines were to prevent `ActiveLearning` from requesting 3 initial runs. Currently they spawn 2 initial runs but don’t do any actual Active Learning.